openlabclient/openlabclient/cli.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import sys
+import yaml
+
+from openlabclient.plugins import base
+from openlabclient import utils
+from openlabclient.utils import _color
+
+
+class OpenLabCmd(object):
+    def __init__(self):
+        self.parser = None
+        self.args = None
+
+    def create_parser(self):
+        parser = argparse.ArgumentParser(
+            description='The command line tool for OpenLab management',
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+        parser.add_argument('-c', dest='config',
+                            default='/etc/openlab/openlab.yaml',
+                            help='path to config file')
+
+        subparsers = parser.add_subparsers(title='commands',
+                                           description='valid commands',
+                                           dest='command',
+                                           help='additional help')
+
+        cmd_check = subparsers.add_parser('check',
+                                          help='check openlab environment')
+        cmd_check.set_defaults(func=self.check)
+        cmd_check.add_argument('--type', default='all',
+                               help="Specify a plugin type, like 'nodepool', "
+                                    "'jobs'. Default is 'all'.")
+        cmd_check.add_argument('--cloud', default='all',
+                               help="Specify a cloud provider, like 'otc', "
+                                    "'vexxhost'. Default is 'all'.")
+        cmd_check.add_argument('--nocolor', action='store_true',
+                               help='Enable the no color mode.')
+        cmd_check.add_argument('--recover', action='store_true',
+                               help='Enable the auto recover mode.')
+
+        return parser
+
+    def _get_cloud_list(self, cloud):
+        with open('/etc/openstack/clouds.yaml') as f:
+            clouds = yaml.load(f, Loader=yaml.FullLoader)
+            clouds_list = [c for c in clouds['clouds']]
+
+        if cloud not in clouds_list + ['all']:
+            print("Error: Cloud %s is not found." % cloud)
+            print("Please use the cloud in %s." % clouds_list)
+            exit(2)
```

Comment:
`sys.exit(main())` in line#109, this is other exit point?

Reply:
Done